website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Connecting your GitLab Account"
+id: "connecting-gitlab"
+sidebar_label: "Connecting your GitLab Account"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, let us know at sales@getdbt.com.
+
+:::
+
+## Overview
+
+Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
+- trigger CI builds when Pull Requests are opened in GitLab!
+- log into dbt Cloud via oauth through GitLab!
+- add new repos to Cloud in a single click!
+- and more!
+
+## Linking dbt Cloud to your GitLab account
+
+To link your dbt Cloud account to your GitLab account, navigate to your Account Settings and click the Integrations tab. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_navigation_setup.gif" title="Like so!"/>
+
+Here you can link your GitLab account to dbt Cloud. 
+
+You'll need to set up an application at GitLab in order to make it official - [they have a guide for that here](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile)! For your dbt Cloud application, you'll want to check the `api` scope level.
```

Comment:
For something like this, I'd outline configurations as a list instead of writing them out in sentence form. Something [like this](https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite#supply-your-oauth-client-id-and-client-secret) makes it easy to follow along with the instructions step-by-step.

I'd also like to see us include:
 - The `name` to use for the integration: `dbt Cloud`
 - The `redirect URI`: copy/paste this from the dbt Cloud Integrations page
 - Select the `confidential` option
 - Select the `api` scope